python/src/trezorlib/toif.py

```diff
@@ -118,15 +122,17 @@ def load(filename: str) -> Toif:
         return from_bytes(f.read())
 
 
-def from_image(image: "Image", background=(0, 0, 0, 255)) -> Toif:
-    if Image is None:
+def from_image(image: "Image", background: Tuple[int, ...] = (0, 0, 0, 255)) -> Toif:
```

Comment:
this is actually `Tuple[int, int, int, int]` (i.e. must be a 4-tuple, not any-tuple)

Reply:
Changed in 4cfe54a